### ISSUE TITLE
PyRevit does not load and is not attached to Revit

### ISSUE THREAD

**Issue by adamjo10** [eirannejad/pyRevit#981]
After installation of Pyrevit it does not show up in Revit 2021. Installed using the .exe installer which completed without errors.

Running `pyrevit attach master 277 --installed` produces error messages that it cannot find clone "master"

Response to pyrevit env as follows:
```
==> Registered Clones (full git repos)
==> Registered Clones (deployed from archive/image)
==> Attachments
==> Installed Extensions
==> Default Extension Search Path
C:\Users\#######\AppData\Roaming\pyRevit\Extensions
==> Extension Search Paths
==> Extension Sources - Default
https://github.com/eirannejad/pyRevit/raw/master/extensions/extensions.json
==> Extension Sources - Additional
==> Installed Revits
2021.1 | Version: 21.1.0.108 | Build: 20200708_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit LT 2021"
2021.1 | Version: 21.1.0.108 | Build: 20200708_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2021"
2019.2.2 Hotfix | Version: 19.2.20.24 | Build: 20190808_0900(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit LT 2019"
==> Running Revit Instances
PID: 24988 | 2021.1 | Version: 21.1.0.108 | Build: 20200708_1515(x64) | Language: 0 | Path: "C:\Program Files\Autodesk\Revit 2021"
==> User Environment
Microsoft Windows 10 [Version 10.0.18363]
Executing User: ######\#######
Active User: ######\#######
Admin Access: No
%APPDATA%: "C:\Users\#######\AppData\Roaming"
Latest Installed .Net Framework: 4.8
No .Net Target Packs are installed.
No .Ne-Core Target Packs are installed.
pyRevit CLI 4.8.1.0
```

Response to `pyrevit attached master 277 -- installed --debug` as follows:

```
<clone_name> = master
attach = True
<engine_version> = 277
--installed = True
Debug: Checking for all: wiki
Debug: Missing: wiki
Debug: Checking for all: blog
Debug: Missing: blog
Debug: Checking for all: docs
Debug: Missing: docs
Debug: Checking for all: source
Debug: Missing: source
Debug: Checking for all: youtube
Debug: Missing: youtube
Debug: Checking for all: support
Debug: Missing: support
Debug: Checking for all: env
Debug: Missing: env
Debug: Checking for all: update
Debug: Missing: update
Debug: Checking for all: clone
Debug: Missing: clone
Debug: Checking for all: clones
Debug: Missing: clones
Debug: Checking for all: attach
Debug: Try getting config as dict "environment:clones"
Debug: Try getting config "environment:clones"
Debug: Config "environment:clones" not set.
Error: Can not find clone "master" (pyRevitLabs.Common.PyRevitException)
   at pyRevitLabs.PyRevit.PyRevitClones.GetRegisteredClone(String cloneNameOrRepoPath)
   at pyRevitCLI.PyRevitCLICloneCmds.AttachClone(String cloneName, Boolean latest, Boolean dynamoSafe, String engineVersion, String revitYear, Boolean installed, Boolean attached, Boolean allUsers)
   at pyRevitCLI.PyRevitCLI.ProcessArguments()
   at pyRevitCLI.PyRevitCLI.Main(String[] args)
 ```

What needs to be done to complete the installation?

**Comment by eirannejad** [comment]
You can manually add the clone using:
`pyrevit clones add master <install path>`

Then run `pyrevit clones` to ensure the clone is added, and run `pyrevit attach master 277 --installed` to attach it to installed Revits

I also see that you have both Revit 2021, and 2021 LT installed?

**Comment by adamjo10** [comment]
I tried the command you suggested with
pyrevit clones add master "C:\Program Files\pyRevit CLI" --debug

The output was as follows:

<clone_name> = master
clones = True
add = True
<clone_path> = C:\Program Files\pyRevit CLI
Debug: Checking for all: wiki
Debug: Missing: wiki
Debug: Checking for all: blog
Debug: Missing: blog
Debug: Checking for all: docs
Debug: Missing: docs
Debug: Checking for all: source
Debug: Missing: source
Debug: Checking for all: youtube
Debug: Missing: youtube
Debug: Checking for all: support
Debug: Missing: support
Debug: Checking for all: env
Debug: Missing: env
Debug: Checking for all: update
Debug: Missing: update
Debug: Checking for all: clone
Debug: Missing: clone
Debug: Checking for all: clones
Debug: Checking for all: info
Debug: Missing: info
Debug: Checking for all: open
Debug: Missing: open
Debug: Checking for all: add
Debug: Normalized as "C:\Program Files\pyRevit CLI"
Debug: Registering clone "C:\Program Files\pyRevit CLI"
Debug: Searching for valid clones above: C:\Program Files\pyRevit CLI
Debug: Normalized as "C:\Program Files\pyRevit CLI"
Debug: Checking pyRevit clone validity "C:\Program Files\pyRevit CLI"
Debug: Checking clone validity by directory structure...
Debug: Normalized as "C:\Program Files\pyRevit CLI\pyrevitlib"
Debug: Normalized as "C:\Program Files\pyRevit CLI\pyrevitlib\pyrevit"
Debug: Checking pyRevit path "C:\Program Files\pyRevit CLI\pyrevitlib\pyrevit"
Debug: Invalid pyRevit clone. | Path "C:\Program Files\pyRevit CLI" is not a valid git pyRevit clone.
Debug: Normalized as "C:\Program Files"
Debug: Checking pyRevit clone validity "C:\Program Files"
Debug: Checking clone validity by directory structure...
Debug: Normalized as "C:\Program Files\pyrevitlib"
Debug: Normalized as "C:\Program Files\pyrevitlib\pyrevit"
Debug: Checking pyRevit path "C:\Program Files\pyrevitlib\pyrevit"
Debug: Invalid pyRevit clone. | Path "C:\Program Files" is not a valid git pyRevit clone.
Debug: Normalized as "C:"
Debug: Checking pyRevit clone validity "C:"
Debug: Checking clone validity by directory structure...
Debug: Normalized as "C:\Users\Adam_O\pyrevitlib"
Debug: Normalized as "C:\Users\Adam_O\pyrevitlib\pyrevit"
Debug: Checking pyRevit path "C:\Users\Adam_O\pyrevitlib\pyrevit"
Debug: Invalid pyRevit clone. | Path "C:" is not a valid git pyRevit clone.
Debug: Searching for valid clones below: C:\Program Files\pyRevit CLI
Debug: Normalized as "C:\Program Files\pyRevit CLI"
Debug: Checking pyRevit clone validity "C:\Program Files\pyRevit CLI"
Debug: Checking clone validity by directory structure...
Debug: Normalized as "C:\Program Files\pyRevit CLI\pyrevitlib"
Debug: Normalized as "C:\Program Files\pyRevit CLI\pyrevitlib\pyrevit"
Debug: Checking pyRevit path "C:\Program Files\pyRevit CLI\pyrevitlib\pyrevit"
Debug: Invalid pyRevit clone. | Path "C:\Program Files\pyRevit CLI" is not a valid git pyRevit clone.
Error: Path does not point to a valid clone "C:\Program Files\pyRevit CLI" (pyRevitLabs.Common.PyRevitException)
   at pyRevitLabs.PyRevit.PyRevitClone..ctor(String clonePath, String name)
   at pyRevitLabs.PyRevit.PyRevitClones.RegisterClone(String cloneName, String repoPath, Boolean forceUpdate)
   at pyRevitCLI.PyRevitCLI.ProcessArguments()
   at pyRevitCLI.PyRevitCLI.Main(String[] args)

It seems it can't find anything to clone.

What did I do wrong?
I downloaded "pyRevit.CLI_4.8.1_signed.exe" from your website
I ran the installer and entered the admin account username and password when required

The software has been installed into the default location "C:\Program Files\pyRevit CLI"

Is there something else that needs to be installed?
What path should I use for the clone?

LT is also installed as we started with that, then got some (much) larger projects and needed the full version.....
